Project.toml

```diff
@@ -0,0 +1,18 @@
+name = "FTPServer"
+uuid = "9c2b0ca7-0fb3-5789-8f1a-7604e426024c"
```

Comment:
Add a version here (check on GitHub releases tab and increment)
Add `author="Invenia Technical ....`